<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionWindow</class>
 <widget class="QMainWindow" name="AcquisitionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EEG信号采集平台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>起始频率(Hz)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>21</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20.5</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>截止频率(Hz)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>21</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>28</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>45</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>陷波(Hz)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>21</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>60</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter">
          <property name="maximumSize">
           <size>
            <width>93</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>滤波</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>手动标记</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>M1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>M2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>M3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>21</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>M4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QChartView" name="graphicsView"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QChartView" name="graphicsView_2"/>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphicsView_3"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据保存</string>
    </property>
    <addaction name="actionEDF"/>
    <addaction name="actionBDF"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>实验选项</string>
    </property>
    <addaction name="actionStart_Recording"/>
    <addaction name="actionStop_Recording"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>各导电压</string>
     </property>
     <addaction name="action_10_10uV"/>
     <addaction name="action_25_25uV"/>
     <addaction name="action50uV"/>
     <addaction name="action100uV"/>
     <addaction name="action200uV"/>
     <addaction name="action_500_500uV"/>
     <addaction name="action_1000_1000uV"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>时间范围</string>
     </property>
     <addaction name="action0_1s"/>
     <addaction name="action0_5s"/>
     <addaction name="action0_10s"/>
    </widget>
    <addaction name="menu_6"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEDF">
   <property name="text">
    <string>edf</string>
   </property>
  </action>
  <action name="actionOther">
   <property name="text">
    <string>from E-prime</string>
   </property>
  </action>
  <action name="actionStart_Recording">
   <property name="text">
    <string>开始记录</string>
   </property>
  </action>
  <action name="actionStop_Recording">
   <property name="text">
    <string>停止记录</string>
   </property>
  </action>
  <action name="actionp300oddball">
   <property name="text">
    <string>oddball</string>
   </property>
  </action>
  <action name="actionBDF">
   <property name="text">
    <string>bdf</string>
   </property>
  </action>
  <action name="action50uV">
   <property name="text">
    <string>50</string>
   </property>
  </action>
  <action name="action100uV">
   <property name="text">
    <string>100</string>
   </property>
  </action>
  <action name="action200uV">
   <property name="text">
    <string>200</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action16">
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action8_2">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action16_2">
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action32">
   <property name="text">
    <string>32</string>
   </property>
  </action>
  <action name="actionBEAM">
   <property name="text">
    <string>绘制地形图</string>
   </property>
  </action>
  <action name="actionBrainNetwork">
   <property name="text">
    <string>脑网络分析</string>
   </property>
  </action>
  <action name="action0_1s">
   <property name="text">
    <string>0~1s</string>
   </property>
  </action>
  <action name="action0_5s">
   <property name="text">
    <string>0~5s</string>
   </property>
  </action>
  <action name="action0_10s">
   <property name="text">
    <string>0~10s</string>
   </property>
  </action>
  <action name="action_10_10uV">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action_25_25uV">
   <property name="text">
    <string>25</string>
   </property>
  </action>
  <action name="action_500_500uV">
   <property name="text">
    <string>500</string>
   </property>
  </action>
  <action name="action_1000_1000uV">
   <property name="text">
    <string>1000</string>
   </property>
  </action>
  <action name="action5000">
   <property name="text">
    <string>5000</string>
   </property>
  </action>
  <action name="action10000">
   <property name="text">
    <string>10000</string>
   </property>
  </action>
  <action name="action50000">
   <property name="text">
    <string>50000</string>
   </property>
  </action>
  <action name="action100000">
   <property name="text">
    <string>100000</string>
   </property>
  </action>
  <action name="action200000">
   <property name="text">
    <string>200000</string>
   </property>
  </action>
  <action name="action400000">
   <property name="text">
    <string>400000</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
